addons/website_slides/views/website_slides_templates_utils.xml

```diff
@@ -110,7 +110,7 @@
         <t t-call="website_slides.slide_share_link"/>
         <h5 class="mt-3">Share on Social Media</h5>
         <t t-call="website_slides.slide_share_social"/>
-        <t t-call="website_slides.slide_social_email"/>
+        <t t-if="record.share_channel_template_id.id" t-call="website_slides.slide_social_email" />
```

Comment:
We could to that in `master` but on stable versions, customers will not benefit from this fix (except if they manually update the module/template)
Considering the existing databases, I would rather fix on PY side
When clicking on "Send Email", it leads to:
https://github.com/odoo/odoo/blob/cc6e4aa8e5d7f0bc4e0479c6f5ab600de56305e4/addons/website_slides/models/slide_channel.py#L780
which will trigger the error because of the empty `template`:
https://github.com/odoo/odoo/blob/e5dbded9bb363351feff7ca8a56c7f8a6860f492/addons/mail/models/mail_template.py#L329
So, why not add a `UserError` saying that we can't send any email because of the missing template

Reply:
Yes  it's better, I will fix it